server.js

```diff
@@ -5,8 +5,6 @@ global.jwt_secret = 'vbbbcvv765656hvhdgAdsFBEDstdsds6725764';
 //Changing this secret will reset all token to false
 server.connection({ port: 8000 ,routes: {cors: true }});
```

Comment:
@tusharncuelogic @sandeepp-cuelogic @gauravc-cuelogic @pritig-cuelogic port number should read from environment variable.